src/components/editor/faultTree/menu/faultEvent/FaultEventMenu.tsx

```diff
@@ -323,11 +323,10 @@ const FaultEventMenu = ({ selectedShapeToolData, onEventUpdated, refreshTree, ro
 
   return (
     <Box paddingLeft={2} marginRight={2}>
-      {shapeToolData && (
-        <ReusableFaultEventsProvider systemUri={selectedSystem?.iri}>
-          <FaultEventShapeToolPane data={shapeToolData} refreshTree={refreshTree} formMethods={formMethods} />
-        </ReusableFaultEventsProvider>
-      )}
+      <ReusableFaultEventsProvider systemUri={selectedSystem?.iri}>
```

Comment:
just note you removed `shapeToolData &&`. Not sure why it was there in the first place.

Reply:
I also not sure why `shapeToolData &&` was there. The component `FaultEventShapeToolPane` is implemented so that it can handle if `shapeToolData` is null by  rendering a message that the there is no selection.